### ISSUE TITLE
Repo with two remotes report a bug

### ISSUE THREAD

**Issue by psychelzh** [eamodio/vscode-gitlens#722]
- GitLens Version: `9.6.2`
  
- VSCode Version: 1.33.1 (system setup)
- OS Version: Windows_NT x64 10.0.17763

When I push all my branches to two remotes, they seems to contradict with each other. I have locally three branches: `master`, `develop` and `feature/analyze-blink-state`.

Error message: 

```
ID 为 0:gitlens:repository(c:/Users/liang/OneDrive/Documents/psychelzh/EyeblinkPro):remote-branch-folder(/feature) 的元素已被注册
```

**Comment by eamodio** [comment]
@psychelzh Unfortunately I don't know what you are reporting -- what is the issue? How are you pushing your branches? What is showing up wrong in GitLens?

**Comment by psychelzh** [comment]
I give one screenshot here. Hope it will make the issue clearer.

![image](https://user-images.githubusercontent.com/13306020/56795029-22a6d580-6842-11e9-992f-d42b01934e9d.png)

**Comment by eamodio** [comment]
@psychelzh I believe this was fixed in GitLens 9.7. Please let me know if you are still seeing this issue. Thanks!

**Comment by psychelzh** [comment]
@eamodio Yeah, it was fixed. Thank you for your effort.
